griptape/utils/observability/griptape_cloud_span_exporter.py

```diff
@@ -0,0 +1,49 @@
+import os
+from collections.abc import Sequence
+from urllib.parse import urljoin
+from typing_extensions import override
+from attrs import Factory, define, field
+from opentelemetry.sdk.trace import ReadableSpan
+from opentelemetry.sdk.trace.export import SpanExporter, SpanExportResult
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+
+
+@define
+class GriptapeCloudSpanExporter(SpanExporter):
+    base_url: str = field(
+        default=Factory(lambda: os.getenv("GT_CLOUD_BASE_URL", "https://cloud.griptape.ai")), kw_only=True
+    )
+    api_key: str = field(default=Factory(lambda: os.getenv("GT_CLOUD_API_KEY")), kw_only=True)
+    headers: dict = field(
+        default=Factory(lambda self: {"Authorization": f"Bearer {self.api_key}"}, takes_self=True), kw_only=True
+    )
+    structure_run_id: str = field(default=Factory(lambda: os.getenv("GT_CLOUD_STRUCTURE_RUN_ID")), kw_only=True)
+    _underlying: SpanExporter = field(
```

Comment:
Is "underlying" an open telemetry term? Can we name it something like `exporter` instead?

Reply:
It comes from some past application of the decorator pattern. Mad naming more consistent in new abstractions.